public/index.html

```diff
@@ -22,7 +22,6 @@
   <link rel="icon" type="image/png" href="%PUBLIC_URL%/images/favicon/favicon-32x32.png" sizes="32x32" />
   <link rel="icon" type="image/png" href="%PUBLIC_URL%/images/favicon/favicon-16x16.png" sizes="16x16" />
   <link rel="mask-icon" href="%PUBLIC_URL%/images/favicon/safari-pinned-tab.svg">
-  <script src="https://accounts.google.com/gsi/client" async defer id="gsiClient"></script>
```

Comment:
Drive by fix - we removed google sign in https://github.com/DataBiosphere/duos-ui/pull/2667